helper/linker.go

```diff
@@ -43,70 +62,139 @@ func (f FileLinker) Execute() (map[string]string, error) {
 		layerDir = "/layers/paketo-buildpacks_open-liberty/open-liberty-runtime"
 	}
 
-	b, ok, err := bindings.ResolveOne(f.Bindings, bindings.OfType("open-liberty"))
-	if err != nil {
-		return nil, fmt.Errorf("error resolving bindings: %w", err)
+	if err = f.Configure(layerDir, appDir); err != nil {
+		return nil, fmt.Errorf("error during configuration: %w", err)
```

Comment:
```suggestion
		return map[string]string{}, fmt.Errorf("error during configuration: %w", err)
```

Same, just trying to keep styles consistent.

Reply:
```suggestion
		return nil, fmt.Errorf("unable to configure\n%w", err)
```

More style stuff. We try to use `unable to ...` with messages. Not my choice, but for consistency, we also use `\n` before printing other errors.

I don't think some of this came from your commit, but using it as a chance to refactor. Pardon the extra noise.